README.md

```diff
@@ -47,321 +47,555 @@ uor-client-go version
 1. Use `uor-client-go build schema` to build a schema to be used with a collection.
 2. Use the `uor-client-go build collection` command to build the workspace as an OCI artifact in build-cache The default location is ~/.uor/cache. It can be set with the `UOR_CACHE` environment variable`.
 3. Use the `uor-client-go push` command to publish to a registry as an OCI artifact.
-4. Use the `uor-client-go pull` command to pull the artifact back to a local workspace. 
+4. Use the `uor-client-go pull` command to pull the artifact back to a local workspace.
 5. Use the `uor-client-go inspect` command to inspect the build cache to list information about references.
 
 ### Build a schema into an artifact
-```
-# This is a optional precursor to building a collection. Collections can reference an
-# already built schema or no schema at all.
+
+A schema can be optionally created prior to building a collection. Collections can then reference an already built schema or no schema at all.
+
+```shell
 uor-client-go build schema schema-config.yaml localhost:5000/myschema:latest
 ```
+
 ### Build workspace into an artifact
 
-```
+Execute the following command to build a workspace into an an artifact:
+
+```shell
 uor-client-go build collection my-workspace localhost:5000/myartifacts:latest
 ```
 
-```
-# Optionally with dsconfig
+An optional dataset configuration file can be provided using the `--dsconfig` flag:
+
+```shell
 uor-client-go build my-workspace localhost:5000/myartifacts:latest --dsconfig dataset-config.yaml
 ```
+
 ### Push workspace to a registry location
 
-```
+Push a workspace to a remote registry
+
+```shell
 uor-client-go push my-workspace localhost:5000/myartifacts:latest
 ```
 
 ### Pull UOR collection to a location
 
-```
+Pull a collection from a remote registry:
+
+```shell
 uor-client-go pull localhost:5000/myartifacts:latest -o my-output-directory
 ```
 
 ### Pull subsets of a UOR collection to a location by attribute
 
-```
+Pull a portion of a collection by filtering for a set of attribute:
+
+```shell
 uor-client-go pull localhost:5000/myartifacts:latest -o my-output-directory --attributes key=value
 ```
 
 ## Getting Started
```

Comment:
We should probably plan on moving technical information into the docs/ path pending resolution of https://github.com/uor-framework/universalreference.io/issues/17

Reply:
Relevant to suggestion https://github.com/uor-framework/universalreference.io/pull/22